insights/parsers/ovs_vsctl_list_bridge.py

```diff
@@ -4,16 +4,28 @@
 
 This module provides class ``OVSvsctlListBridge`` for parsing the
 output of command ``ovs-vsctl list bridge``.
+
+.. note::
+    Please refer to its super-class :class:`insights.parsers.ovs_vsctl_list.OVSvsctlList`
+    for more details.
+
+.. warning::
+    This parser `OVSvsctlListBridge` is deprecated, please use
+    :py:class:`insights.ovs_vsctl.OVSvsctlListBridge` instead.
 """
 
-from insights import CommandParser, get_active_lines, parser
-from insights.parsers import SkipException, optlist_to_dict
+from insights import parser
+from insights.parsers.ovs_vsctl import OVSvsctlList
```

Comment:
You may want to use the `util.deprecated` function to mark this parser will be deprecated in the future version , see  https://github.com/RedHatInsights/insights-core/blob/master/insights/parsers/journal_since_boot.py#L55

Reply:
@xiangce Thanks, Updated it, please have a look.